mediapipe/tasks/ios/test/audio/core/utils/sources/AVAudioPCMBuffer+TestUtils.m

```diff
@@ -0,0 +1,88 @@
+// Copyright 2024 The MediaPipe Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "mediapipe/tasks/ios/test/audio/core/utils/sources/AVAudioPCMBuffer+TestUtils.h"
+
+@implementation AVAudioPCMBuffer (TestUtils)
+
+- (nullable AVAudioPCMBuffer *)initFromAudioFileWithInfo:(MPPFileInfo *)fileInfo
```

Comment:
Sorry - this doesn't pass our internal checks. I think we need to make this a static factory as all initializers must call "sell init".